include/tim/lite/handle.h

```diff
@@ -32,9 +32,16 @@ namespace lite {
 
 class HandleImpl;
 
+enum class HandleFlushType {
+    HandleFlush = 0,
+    HandleInvalidate = 1
+};
+
```

Comment:
If you want to keep this difinition, then move them to src not in the public api.

Reply:
Have updated.